ibis/pandas/core.py

```diff
@@ -152,6 +166,10 @@ def execute_with_scope(expr, scope, aggcontext=None, clients=None, **kwargs):
     scope : collections.Mapping
         A dictionary mapping :class:`~ibis.expr.operations.Node` subclass
         instances to concrete data such as a pandas DataFrame.
+    timecontext : TimeContext
```

Comment:
Optional

Reply:
Just realized that in definition TimeContext is already Optional. I have made all timecontext typings match with each other.